doc/docs/Python_User_Interface.md

```diff
@@ -4156,6 +4156,85 @@ media](FAQ.md#why-does-my-simulation-diverge-if-0).
 
 </div>
 
+---
+<a id="MaterialGrid"></a>
+
+### MaterialGrid
+
+```python
+class MaterialGrid(object):
+```
+
+<div class="class_docstring" markdown="1">
+
+This class is used to specify materials on a discrete Cartesian grid. A class object is passed as the `material`
+argument of a [`GeometricObject`](#GeometricObject) or the `default_material` argument of the `Simulation`
+constructor (similar to a material function).
+
+</div>
+
+
+
+<a id="MaterialGrid.__init__"></a>
+
+<div class="class_members" markdown="1">
+
+```python
+def __init__(self,
+             grid_size,
+             medium1,
+             medium2,
+             design_parameters=None,
+             grid_type='U_DEFAULT',
+             do_averaging=False,
+             beta=0,
+             eta=0.5):
+```
+
+<div class="method_docstring" markdown="1">
+
+Creates a `MaterialGrid` object.
+
+The input are two materials `medium1` and `medium2` which are linearly interpolated at each grid point using
+a NumPy array `design_parameters` of size `grid_size` (a 3-tuple or `Vector3` of integers) with floating-point values in
+the range [0,1] used to define a linear weight (i.e., 0 is `medium1` and 1 is `medium2`). Currently, only two material
+types are supported: (1) frequency-independent isotropic $\varepsilon$ or $\mu$ and (2) `LorentzianSusceptibility`.
+`medium1` and `medium2` must both be the same type.
+
+[Subpixel smoothing](Subpixel_Smoothing.md) can be enabled by specifying `do_averaging=True`. To improve the accuracy
```

Comment:
This is not to "improve the accuracy".   It's just different.   I would say something like:

If you want to use a material grid to define a (nearly) discontinuous, piecewise-constant material that is *either* `medium1` or `medium2` almost everywhere, you can optionally enable a (smoothed) *projection* feature by setting the parameter `beta` to a positive value.